### ISSUE TITLE
Split button action dropdown: improve keyboard interaction

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly-react#3337]
**Describe the issue. What is the expected and unexpected behavior?**
Currently in the split button action dropdown there are two buttons without interaction (i.e. if you tab to the action button, pressing enter or space does nothing, you would have to tab again, over to the toggle button beside it.) From a keyboard interaction perspective, having a button without interaction is confusing for keyboard users. 

The design for having both buttons is to have the primary action exposed as a button to save the need to open the menu. It also provides better visibility to the primary action. However, it would be expected that the user could initiate that action as they would for a standard button (enter or space). If they want to open that menu, they would tab over to the toggle.

So, the change here would be to enable a keyboard trigger on the button. Perhaps show that interaction on the example. It should be clear that the primary action is exposed and that the button element was used intentionally.

**Please provide the steps to reproduce. Feel free to link CodeSandbox or another tool.**

**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**
An enhancement to the keyboard interaction and clarifying the primary action button.

**What is your product and what release version are you targeting?**
n/a
